cachecontrol/caches/redis_cache.py

```diff
@@ -26,9 +26,24 @@ def set(self, key, value, expires=None):
         if not expires:
             self.conn.set(key, value)
         else:
-            expires = expires - datetime.utcnow()
-            self.conn.setex(key, total_seconds(expires), value)
-
+            # the keyword arguments are to account for a Redis v StrictRedis issue
+            # with pyredis being a mess. this is compatible with both.
+
+            redis_class = self.conn.__class__
```

Comment:
Why can't we use kwargs instead of positional args regardless on Redis vs StrictRedis? E.g:

```self.conn.setex(key=key, expires=expires, value=value)```